test/jdk/java/util/Locale/ThaiGov.java

```diff
@@ -20,98 +20,70 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-/**
+
+/*
  * @test
  * @bug 4474409
+ * @summary Tests some localized methods with Thai locale
  * @author John O'Conner
  * @modules jdk.localedata
- * @run main/othervm -Djava.locale.providers=COMPAT ThaiGov
+ * @run junit/othervm -Djava.locale.providers=COMPAT ThaiGov
  */
 
-import java.util.*;
-import java.text.*;
-
-public class ThaiGov {
+import java.text.DateFormat;
+import java.text.NumberFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
 
-    ThaiGov() {
-        System.out.println("ThaiGov locale test...");
+import org.junit.jupiter.api.Test;
 
-    }
-
-    void numberTest() throws RuntimeException {
-        final String strExpected = "\u0E51\u0E52\u002C\u0E53\u0E54\u0E55\u002C\u0E56\u0E57\u0E58\u002E\u0E52\u0E53\u0E54";
-        final double value =  12345678.234;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-        Locale locTH = Locale.of("th", "TH", "TH");
-
-        // th_TH_TH test
-        NumberFormat nf = NumberFormat.getInstance(locTH);
-        String str = nf.format(value);
+public class ThaiGov {
 
-        if (!strExpected.equals(str)) {
-            throw new RuntimeException();
-        }
+    private static final double VALUE = 12345678.234;
+    private static final Locale TH = Locale.of("th", "TH", "TH");
 
+    // Test number formatting for thai
+    @Test
+    public void numberTest() throws RuntimeException {
```

Comment:
Not your change, but this `throws` is redundant

Reply:
Thanks for the review, addressed this and your other suggestions as well.